RNTester/js/PanResponderExample.js

```diff
@@ -10,29 +10,27 @@
 
 'use strict';
 
-var React = require('react');
-var createReactClass = require('create-react-class');
-var ReactNative = require('react-native');
-var {PanResponder, StyleSheet, View} = ReactNative;
+const React = require('react');
+const ReactNative = require('react-native');
+const {PanResponder, StyleSheet, View} = ReactNative;
 
-var CIRCLE_SIZE = 80;
+const CIRCLE_SIZE = 80;
 
-var PanResponderExample = createReactClass({
-  displayName: 'PanResponderExample',
+class PanResponderExample extends React.Component<{}> {
+  static displayName: ?string = 'PanResponderExample';
 
   statics: {
     title: 'PanResponder Sample',
-    description:
-      'Shows the use of PanResponder to provide basic gesture handling.',
-  },
+    description: 'Shows the use of PanResponder to provide basic gesture handling.',
+  };
```

Comment:
Ha! Flow makes this invalid JavaScript valid. Your code currently translates to this: 

```typescript
type T = {
  title: 'PanResponder Sample',
  description: 'Shows the use of PanResponder to provide basic gesture handling.',
}

class PanResponderExample extends React.Component<{}> {
  // ...
  statics: T;
  // ...
}
```

I think what you really mean is this:

```JavaScript
static title = 'PanResponder Sample',
static description = 'Shows the Use of PanResponder to provide basic gesture handling',
```

Reply:
👍